ts/features/bonus/bpd/components/transactionItem/__test__/BdpTransactionItem.test.tsx

```diff
@@ -0,0 +1,56 @@
+// import * as React from "react";
+// import { render } from '@testing-library/react-native';
+import { getSubtitle } from "../BpdTransactionItem";
+import { EnhancedBpdTransaction } from "../../../components/transactionItem/BpdTransactionItem";
+import { HPan } from "../../../store/actions/paymentMethods";
+import { AwardPeriodId } from "../../../store/actions/periods";
+
+describe("Test how the transaction subtitle changes with different timestamps", () => {
+  it("Subtitle shouldn't contain hours and minutes, when the transaction has a timestamp 00:00", () => {
+
+    // TODO: test  input data should be moved in a separate file
```

Comment:
TODO should be followed by a ref to a story (es https://github.com/pagopa/io-app/blob/0e30730f595b2b041b667fc88cf45fd123ffd51f/ts/components/screens/BadgeComponent.tsx#L3) or Can we remove it?

Reply:
This is an old comment, I don't know if I can remove it

Reply:
I'll remove the TODO.. I don't know if the team agrees on having a separate module for test input data, this is just my approach